source/MapSalesPanel.cpp

```diff
@@ -128,6 +131,11 @@ bool MapSalesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 
 bool MapSalesPanel::Click(int x, int y, int clicks)
 {
+	static const int KEY_ROW_COUNT = 4;
+
+	const Interface *ui = GameData::Interfaces().Get("sales key");
+	const Rectangle keyContentBox = ui->GetBox("content");
```

Comment:
There is a `UI` class, so please rename this variable. Maybe something like this:
```suggestion
	const Interface *keyInterface = GameData::Interfaces().Get("sales key");
	const Rectangle keyContentBox = keyInterface->GetBox("content");
```